lighthouse-core/report/html/report-styles.css

```diff
@@ -953,13 +949,14 @@ summary.lh-passed-audits-summary {
   animation: fadeInTooltip 150ms;
   animation-fill-mode: forwards;
   animation-delay: 900ms;
-  min-width: 15em;
+  min-width: 23em;
```

Comment:
just felt right?

Reply:
Yup. In the new audit-error case the tooltip boundary is super narrow so the tooltip was being displayed at its min-width. 

Behold!

![image](https://user-images.githubusercontent.com/39191/39457370-647dac52-4ca2-11e8-8093-07305ba79806.png)